css/properties/offset-path.json

```diff
@@ -230,6 +230,46 @@
               "deprecated": false
             }
           }
+        },
+        "url": {
+          "__compat": {
+            "description": "<code>&lt;url()</code>",
```

Comment:
```suggestion
            "description": "<code>url()</code>",
```

Reply:
Oops, I missed the right angular bracket. It should be `<url()>`. Thanks for reviewing @queengooborg!